lib/dogma/formatter/emacs.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Dogma.Formatter.Emacs do
```

Comment:
Honestly I was just copying the name from [`rubocop`](https://github.com/bbatsov/rubocop#formatters). I just now poked around the Syntastic checkers, and rubocop was the only one I found that called it `emacs`. For a lot of programs it seems like a similar format is just the default. Others call it `verbose` or `machine-readable`.

We could call it whatever, maybe `machine`?. We could call it `Vim` or `Syntastic` since that seems to be our intended use case. ;)